src/js/BookReader.js

```diff
@@ -581,6 +581,21 @@ BookReader.prototype.drawLeafs = function() {
   }
 };
 
+BookReader.prototype.leafTemplate = function(styles, index) {
```

Comment:
Let's place the index first, and makes styles optional. I find it gets a little lost when reading downstream code.